logos/src/logos/main.py

```
@@ -0,0 +1,416 @@
+    with DBManager() as db:
+        return db.update_policy(**data.dict())
+
+
+@app.post("/logosdb/delete_policy")
+async def delete_policy(data: DeletePolicyRequest):
+    with DBManager() as db:
+        return db.delete_policy(**data.dict())
+
+
+@app.post("/logosdb/get_policy")
+async def add_model(data: GetPolicyRequest):
```

Ruff (0.11.9)

Redefinition of unused `add_model` from line 197

(F811)